data_services/deploy/ansible/roles/forecastio/files/data_queues/forecastio/forecastio_air.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import absolute_import
```

Comment:
Yeah, there is no reason they absolutely have to be together.  I was trying to separate out the various data collectors so they could be cherry-picked as roles.  It may not be worthwhile though.  I am okay with pulling the full django app from a different repo in one piece rather than breaking it apart.